components/ProductInfo/ProdImage.js

```diff
@@ -42,12 +41,20 @@ export default ProdImage;
 
 const ImgBtn = ({ image }) => {
   const dispatch = useDispatch();
-  const displayImg = useSelector(({ prodInfo }) => prodInfo.visibleImage);
+  const {  imgArray, currentImage } = useSelector(({ prodInfo }) => ({
+    imgArray: prodInfo.imageBtnArr,
+    currentImage: prodInfo.currentImage
+  }));
 
   return (
     <button
-      className={`${style.btn} ${displayImg === image && style.active}`}
-      onMouseEnter={() => dispatch(changeVisibleImage(image))}
+      className={`${style.btn} ${currentImage.url === image && style.active}`}
+      onMouseEnter={() => {
+        dispatch(changeVisibleImage(image));
+        dispatch(
+          changeCurrentImage({ url: image, isProd: image !== imgArray[1] })
+        );
+      }}
```

Comment:
on mouse enter change the current state to the image, if its the shirt sizing chart then, set isProd (isProduct) to false